packages/client-api-schema/src/types.ts

```diff
@@ -46,7 +46,11 @@ interface JsonRpcNotification<NotificationName, NotificationParams> {
   params: NotificationParams;
 }
 
-interface JsonRpcError<Code, Message, Data = undefined> {
+export const UserDeclinedErrorCode = 200;
```

Comment:
I ran into `TS1209: Ambient const enums are not allowed when the '--isolatedModules' flag is provided.` in `web3-torrent` when trying to use an enum here, so I just used simple `const` variables.